examples/pykokkos/from_array.py

```diff
@@ -16,17 +16,23 @@ def addition_cp(i: int, cp_arr: pk.View2D[int]):
 
 np_arr = np.zeros((size, 2)).astype(np.int32)
 cp_arr = cp.zeros((size, 2)).astype(np.int32)
+list_arr = [np.array([0, 0], dtype=np.int32)] * size
```

Comment:
this is an example that we want to keep to show that explicit functions can be used, right?

If yes, we should have a corresponding example that looks more or less the same, but uses only from

Reply:
but we do not want explicit functions